docker-compose.yml

```diff
@@ -2,21 +2,24 @@
 version: "2"
 services:
   # To test with MariaDB, set FLEET_MYSQL_IMAGE to mariadb:10.6 or the like.
+  # To run in macOS M1, set FLEET_MYSQL_IMAGE=arm64v8/mysql:oracle FLEET_MYSQL_PLATFORM=linux/arm64v8
```

Comment:
This should be: `FLEET_MYSQL_IMAGE=arm64v8/mysql:oracle FLEET_MYSQL_PLATFORM=linux/arm64/v8`

Other we would get this error:
```Error response from daemon: image with reference arm64v8/mysql:oracle was found but does not match the specified platform: wanted linux/arm64v8, actual: linux/arm64/v8```